tests/services/auth.service.test.js

```diff
@@ -106,31 +97,21 @@ describe('Auth Service', () => {
                 email: 'jon.doe@email.com',
                 password: 'password1',
             }
-            const expectedResult = {
+            const stubUser = {
```

Comment:
When deciding variable names, try to pick a word that reflects what the thing is in the real world. This is case the object represents a `user`